RecoLocalCalo/Configuration/python/hcalLocalRecoNZS_cff.py

```diff
@@ -22,11 +22,18 @@
 import RecoLocalCalo.HcalRecProducers.HBHEPhase1Reconstructor_cfi
 _phase1_hbherecoMB = RecoLocalCalo.HcalRecProducers.HBHEPhase1Reconstructor_cfi.hbheprereco.clone(
     dropZSmarkedPassed = cms.bool(False),
-    recoParamsFromDB = cms.bool(False),
     algorithm = dict(
         useM2 = cms.bool(False),
         useM3 = cms.bool(False)
     ),
+    setNegativeFlagsQIE8 = cms.bool(False),
+    setNegativeFlagsQIE11 = cms.bool(False),
+    setNoiseFlagsQIE8 = cms.bool(True),
```

Comment:
Just to check (as I'm not sure I fully understood the logic in the HBHEPhase1Reconstructor): shouldn't this setNoiseFlagsQIE8 be set to "False", as the other bools here, if you want to retrieve the info from DB (and therefore not to look it up in the constructor) ?